src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,6 @@
+:root {
+  --animation-duration-header: 300ms;
+  --anim-time-cat: 300ms;
```

Comment:
What is cat? My first thought was that it's about cat as animals. Let's give proper names. For example, --animation-duration-card/catalog/sth else.